pytest_fixtures/component/architecture.py

```diff
@@ -1,14 +1,15 @@
 # Architecture Fixtures
 import pytest
-from nailgun import entities
 
 from robottelo.constants import DEFAULT_ARCHITECTURE
 
 
 @pytest.fixture(scope='session')
-def default_architecture():
+def default_architecture(default_sat):
```

Comment:
all uses of the default_Sat fixture must be replaced with `<scope>_target_sat` after the merge of #9562